### ISSUE TITLE
bot operating not very human

### ISSUE THREAD

**Issue by HiddevH** [disarmyouwitha/bobber_bot#5]
I think in order to be more realistic, the bot should have 'humane' respone times, e.g. waiting 0.5-1.5 seconds before clicking the bobber, waiting 0.5-1 seconds before throwing the bobber (I'm not sure how they scan for botting but I assume having unrealistic reaction speed is one thing).

**Comment by disarmyouwitha** [comment]
I agree it could be written to look more human.. I definitely had people messing with my bot:
- trying to stand in my view (that was a good one), 
- trying to swim in the water next to the bobber, 
- trying to fish next to me (I had a problem with earlier implementations before checking the tooltip where it would right-click other people's bobbers -- and attack them if they were opposing faction xD)

So.. people definitely think something is weird about standing there fishing all day...

I've seen other bots that put in a random delay; Log out intermittently; or Log out when /tell was received. I was honestly testing the water (no pun intended xD) to see if I would get banned, but I never had any problems. 

After running this for a while.. if I were trying to catch a fishing bot.. I think it would be easiest at the Auction House, because I was listing an insane amount of fish or things that were made from fish. (Oily Blackmouth/oil; Nightfin snapper/soup) and I was using a character named bobberbot. xD 

I made over 1000g before I quit.. not sure if I was lucky or what, but I certainly don't think there is anything wrong with erring on the side of caution if you wanted to add delays!
